README.md

```diff
@@ -29,11 +36,12 @@ You can use Google Cloud or AWS ECR to host repositories.
 
 #### Pushing development images to AWS Elastic Container Registry
 1. Install the [AWS CLI](https://docs.aws.amazon.com/cli/latest/userguide/installing.html)
-2. Install [jq](https://stedolan.github.io/jq/download/)
-3. Set `AWS_ACCESS_KEY_ID`, `AWS_SECRET_ACCESS_KEY` and `AWS_DEFAULT_REGION` variables
+2. Install [jq](https://stedolan.github.io/jq/download/) (`brew install jq` on Mac)
+3. Install 
+4. Set `AWS_ACCESS_KEY_ID`, `AWS_SECRET_ACCESS_KEY` and `AWS_DEFAULT_REGION` variables
    * you may want to use [`aws-vault exec <profile> bash`](https://github.com/99designs/aws-vault)
      to store credentials or assume role in MFA scenarios.
-4. Create two ECR repositories
+5. Create two ECR repositories
```

Comment:
This feels a little incomplete? maybe link out to instructions on how to do this?